src/store/participants/reducer.js

```diff
@@ -0,0 +1,19 @@
+import { types } from './'
+
+const initialState = {
+  currentParticipant: {},
+  participants: [],
+}
+
+export default (state = initialState, { type, payload }) => {
```

Comment:
Is it ok from design perspective to keep both `participants` and `currentParticipant` in one reducer? Right now it's a bit weird that to get to the participants array for instance while connecting to store in a component you have to refer to `state.participants.participants`